js/components/contactList/contactList_controller.js

```diff
@@ -23,24 +41,41 @@ app.controller('contactlistCtrl', ['$scope', '$filter', '$route', '$routeParams'
 	$scope.$watch('ctrl.routeParams.uid', function(newValue) {
 		if(newValue === undefined) {
 			// we might have to wait until ng-repeat filled the contactList
-			if(ctrl.contactList.length > 0) {
+			if(ctrl.contactList && ctrl.contactList.length > 0) {
```

Comment:
shouldn't be necessary, because of [line 9](https://github.com/owncloud/contacts/pull/65/files#diff-958b5eb8fc0357258de1537b5fd654adR9)